### PR TITLE
Feature/Bug fix if file does not exist

### LINKED ISSUE #159: Empty photo/video selection if file doesn't exist
If there is files paths don't exist selection is just white. So it might be confusing for a user. Not sure exactly why `file.exits()` returns false, but it turns out it's a possible case.

Possible solutions:
- some error icon instead of not loading preview at all
- check file existence before filling `MediaAttachment`

screen
```java
public void bind(Attachment attachment, final OnItemClickListener listener) {
    if (file.exists()) {
 	//load image       
    } else {
        Log.d(TAG, "There is no image from this path");
    }
}
```

### PR DESCRIPTION
**Pull request by adrian09h**
# Submit a pull request

close #159

### CODE REVIEW

**Comment by elevenetc** [comment]
Mon_Oct_21_11:21:44_CEST_2019

File: library/src/main/java/com/getstream/sdk/chat/utils/MessageInputClient.java [hunk 1]
```diff
@@ -143,13 +143,19 @@ private void configSelectAttachView(boolean isMedia, List<Attachment> editAttach
...
+                if (!attachments.isEmpty()){
+                    mediaAttachmentAdapter = new MediaAttachmentAdapter(context, attachments, position -> {
+                        Attachment attachment = attachments.get(position);
+                        attachment.config.setSelected(!attachment.config.isSelected());
+                        mediaAttachmentAdapter.notifyItemChanged(position);
+                        updateComposerViewBySelectedMedia(attachments, attachment);
+                    });
+                    binding.rvMedia.setAdapter(mediaAttachmentAdapter);
+                }else{
+                    Utils.showMessage(context, "There is no photo or video.");
```
Comment (elevenetc) [inline library/src/main/java/com/getstream/sdk/chat/utils/MessageInputClient.java]: Could you also put strings into `strings.xml`? Sooner or later we will need to align texts with other mobile platforms and provide way to change/translate texts for developers. And that's just easier to do from the beginning, rather that later trying to extract all strings.

Comment (adrian09h) [reply library/src/main/java/com/getstream/sdk/chat/utils/MessageInputClient.java]: added code to check file existing when getting files from gallery.

**Comment by adrian09h** [comment]
> Tested it and it doesn't really fix #192 . You can reproduce it with creating empty file:
> 
> ```
> adb shell
> cd /storage/emulated/0
> touch empty.png
> ```
> 
> I don't think that we need to fix it with some code changes. File/Image viewers display such images, but with "unknown preview" (see screenshot), and when user tries to open the app shows error/empty screen.

added code to check file existing when getting files from gallery.

**Pull request merged**